traveler-ios-ui/TravelerKitUI/ModelExtensions/WishlistToggleError+LocalizedError.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  WishlistToggleError+LocalizedError.swift
+//  TravelerKitUI
+//
+//  Created by Ben Ruan on 2019-09-16.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import Foundation
+import TravelerKit
+
+extension WishlistToggleError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .unidentifiedTraveler:
+            return NSLocalizedString("You have to login first to wishlist items.", comment: "User not logged ")
+        case .notInWishlist:
```

Comment:
I think we should rename this case...`notWishlisted` sounds better

Reply:
TBD

Reply:
`notInWishlist` indicates that the user is trying to remove something that's wishlisted but not in current fetched `wishlist`.

Reply:
The whole .notInWishlist is removed because if local result doesn't match remote result we can just fetch our local result again from the beginning.